build.rs

```diff
@@ -107,9 +107,9 @@ fn main() {
                 "cargo:rustc-flags=-L{} -lv8 -lv8_monolith_{} -ldl -lc",
                 output_dir, *PROFILE
             );
-            println!("cargo:rustc-cdylib-link-arg=-Wl,-Bstatic");
-            println!("cargo:rustc-cdylib-link-arg=-lstdc++");
-            println!("cargo:rustc-cdylib-link-arg=-Wl,-Bdynamic");
+            println!(
+                "cargo:rustc-link-lib=stdc++",
```

Comment:
I believe this will prefer dynamic linking which is not what we want.

Reply:
It can't prefer anything, since the artifact is a static library, it can only link static libraries into it:

```
❯ file target/debug/libv8_rs.rlib
target/debug/libv8_rs.rlib: current ar archive

❯ ldd target/debug/libv8_rs.rlib
ldd: warning: you do not have execution permission for `target/debug/libv8_rs.rlib'
	not a dynamic executable
```
The rust library crates are always statically linked.

If you want, I can be more specific and specify the static linkage:
```
cargo:rustc-link-lib=-l static:stdc++
```